website/docs/07-apps/01-included-in-onion/pico8-wrapper.md

```diff
@@ -0,0 +1,115 @@
+---
+slug: /apps/pico8-wrapper
+description: PICO-8 Native Wrapper by XK9274
+---
+
+:::caution 
+This section refers to an application planned for inclusion in a future Onion release. Final instructions and usage details may differ from the ones described here. 
+Check [here](https://www.reddit.com/r/MiyooMini/comments/17nw79r/announcing_native_pico8_wrapper_for_onion_with/k7ueyec/?context=3) for playtesting details.
+:::
+
+# PICO-8 Native Wrapper / Splore
+
+The PICO-8 Native Wrapper allows you to play PICO-8 without the need for emulation (hence providing full compatibility with all PICO-8 games), and grants access to Splore, a "frontend" app that allows you to browse, search and play through the entirety of the PICO-8 catalogue (as indexed by the [Lexaloffle BBS](https://www.lexaloffle.com/bbs/)), as long as you have an internet connection. 
+
+Splore also allows you to navigate through and play local cartridge files.
+
+This is an alternative to the RetroArch `fake-08` core, which has access to all typical RetroArch features such as rewinding, fast forward and save state management, but can have compatibility with a portion of the PICO-8 library, specially recently released games.
+
+## How to Enable and Launch
+
+1. Enable the `PICO-8 Native Wrapper` App using the Onion Package Manager's *Expert* section.
+
+2. In order to use the wrapper, you need to have previously purchased PICO-8. It is available as a single purchase at [Lexaloffle's website](https://www.lexaloffle.com/games.php?page=updates). Once downloaded, place the `pico8_dyn` and `pico8.dat` files from the **Raspberry Pi** package in the `/App/pico/bin` directory of your Onion SD card.

```

Comment:
- These binaries will need to be added to: /mnt/SDCARD/RApp/PICO-8/bin
- Any saves and config will go to: /mnt/SDCARD/Saves/CurrentProfile/saves/PICO-8
- Carts will follow the default Onion path: /mnt/SDCARD/Roms/PICO
- Splore carts will be stored in: /mnt/SDCARD/Roms/PICO/splore
- Favourites.txt (if you want to move it) should go in: /mnt/SDCARD/Saves/CurrentProfile/saves/PICO-8/.lexaloffle/pico-8/